src/index.html

```diff
@@ -9,6 +9,54 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a class="header__logo" href="#home">
+        <img 
+          class="logo" 
+          src="images/logo.png" 
+          alt="logo"
+        >
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a class="nav__link is-active" href="#">Apple</a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#">Samsung</a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#">Smartphones</a>
+          </li>
+          
+          <li class="nav__item">
+            <a 
+              class="nav__link" 
+              data-qa="hover" 
+              href="#">Laptops & Computers
+            </a>
```

Comment:
please pay attention to spaces
```suggestion
              href="#"
            >
              Laptops & Computers
            </a>
```